book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] The :rental needs to be getter and setter as well, please add it under attr_accessor

Reply:
@GzimAsani,
thanks for your good and valuable feedback,
I have considered your comment into account.